core/importer/partners_exporter.py

```diff
@@ -0,0 +1,54 @@
+import json
+import sys
+
+from core.models import Partner
+from core.utils import DaisyLogger
+from django.conf import settings
+from io import StringIO
+
+logger = DaisyLogger(__name__)
+
+
+class PartnersExporter:
+
+    def export_to_file(self, file_handle, stop_on_error=False, verbose=False):
+        result = True
+        try:
+            buffer = self.export_to_buffer(StringIO())
```

Comment:
I would either pass the file_handle to the export_to method or have this method returns a string